image_server/server_setup.md

```diff
@@ -48,22 +54,29 @@ It takes about 5 minutes to stand up the instance and load balancer.
 
 1. SSH to the instance (IP address can be found in the EC2 Instance information under Public IP4)
 
-1. Clone the [gis-services github repository](https://github.com/ASFHyP3/gis-services/) to `/home/ubuntu/` on the server
+2. Clone the [gis-services github repository](https://github.com/ASFHyP3/gis-services/) to `/home/ubuntu/` on the server
 ```
 cd /home/ubuntu/
 git clone https://github.com/ASFHyP3/gis-services/
 cd gis-services
-# check out the `develop` branch if on a test server; check out the `main` branch if on a production server
+```
+Make sure to check out the `develop` branch if on a test server; check out the `main` branch if on a production server.
+```
 # git checkout main
 # git checkout develop
 ```
 
-2. Run the server setup script
+3. Run the server setup script, which prompt you for the sitadmin and asf_publisher passwords, which can be found in the tool user account's AWS Secrets Manager
 ```
 /home/ubuntu/gis-services/image_server/server_setup.sh
 ```
+Note that after this runs, the server will be re-booted. It might take a couple of minutes to be able to log into the server again after the restart. 
+
+4. Add any needed public keys to `/home/ubuntu/.ssh/authorized_keys` so that other Tools team members can ssh to the server.
 
-3. Add any needed public keys to `/home/ubuntu/.ssh/authorized_keys` so that other Tools team members can ssh to the server
+5. Schedule scripts to run using crontab with `sudo crontab -e`. Each script can be set up to run at a specific time by specifying the minute, hour, day, and month, and day of the week. For example, to run `make_rtc_services.sh` at midnight and noon every day with a log output added to `make_rgb_service.log`, the command should be written as: 
+`0 0,12 * * * script -qef -c "/home/ubuntu/gis-services/image_services/rtc_services/make_rtc_service.sh /home/ubuntu/gis-services/image_services/rtc_services/nasa_disasters /home/ubuntu/gis-services/image_services/rtc_services/nasa_disasters/rgb.json" -a /home/ubuntu/gis-services/image_services/rtc_services/nasa_disasters/make_rgb_service.log`
+When deciding when to run each script, be mindful of how long running each script might take to avoid publishing multiple services at once. 
```

Comment:
I added `crontab.txt` to this PR and added a command to `server_setup.sh` to install it, do you feel that's sufficient?

The script would install it on any new server, whether test or prod. On test servers I imagine we might immediately comment out many of the entries, though. Thoughts?

Reply:
Maybe tweak the first sentence to say "review and revise the default crontab as needed with `crontab -e`"? The rest of the content looks great and adds helpful context.

No need for `sudo` here, all the crontab entries are owned by `ubuntu` in the new server setup.

Reply:
I think that is a good idea to re-write with review and revise using crontab -e! I'll make that change